CONTRIBUTING.md

```diff
@@ -26,13 +26,12 @@ Symfony projects for instance).
 * Create a new branch, e.g. `feature-foo` or `bugfix-bar`.
 * Make changes.
 * If you are adding functionality or fixing a bug - add a test! Prefer adding new test cases over modifying existing ones.
-* Make sure there is no wrong file permissions in the repository: `./dev-tools/check_file_permissions.sh`.
-* Make sure there is no trailing spaces in the code: `./dev-tools/check_trailing_spaces.sh`.
-* Update documentation: `php dev-tools/doc.php`. This requires the highest version of PHP supported by PHP CS Fixer. If it is not installed on your system, you can run it in a Docker container instead: `docker run -it --rm --user="$(id -u):$(id -g)" -w="/app" --volume="$(pwd):/app" php:7.4-cli php dev-tools/doc.php`.
-* Install dev tools: `dev-tools/install.sh`
-* Run static analysis using PHPStan: `php -d memory_limit=256M dev-tools/vendor/bin/phpstan analyse`
-* Check if tests pass: `vendor/bin/phpunit`.
-* Fix project itself: `php php-cs-fixer fix`.
+* Make sure there is dev environment is correct: `composer dev-tools:check`.
+* Update documentation: `composer dev-tools:docs`. This requires the highest version of PHP supported by PHP CS Fixer. If it is not installed on your system, you can run it in a Docker container instead: `docker run -it --rm --user="$(id -u):$(id -g)" -w="/app" --volume="$(pwd):/app" php:7.4-cli php dev-tools/doc.php`.
```

Comment:
FYI: Docker command will change in #6833 so I did not change it here since official `php:7.4-cli does not include Composer. Depending on which gets merged first I'll alter PR to use proper command here.